src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -1682,5 +1329,30 @@ public TestFormattingOptionsProvider(FormattingOptions options)
 
             public override FormattingOptions GetOptions(LSPDocumentSnapshot documentSnapshot) => _options;
         }
+
+        private class TestTextStructureNavigatorSelectorService : ITextStructureNavigatorSelectorService
```

Comment:
I didn't like that we previously mocked the NavigatorSelectorService so I un'mocked it and created a test class instead